api/types/types.go

```diff
@@ -204,6 +204,7 @@ type Info struct {
 	RuncCommit         Commit
 	InitCommit         Commit
 	SecurityOptions    []string
+	ProductLicense     string
```

Comment:
For Enterprise Engines; is a single string all the information we need? (e.g. expiry date of the license, type of license and such?)

Reply:
Covered by the licensing library - see https://github.com/docker/licensing/blob/master/model/subscriptions.go#L38

I'm working on a change for the enterprise engine to wire it up and hope to have that posted in the next day or two.

(I think we'll likely refine the exact string a bit as we finalize UX but the intention is a human friendly representation of the license)